tests/expected/concat-tests/ember-tests.js

```diff
@@ -1,141 +1,163 @@
-enifed("ember-metal/binding", ["exports"], function (exports) {
-  "use strict";
-});
+//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImVtYmVyLW1ldGFsL2FycmF5LmpzaGludC5qcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiOzs7QUFBQSxTQUFPLHNCQUFQO0FBQ0EsT0FBSyx5Q0FBTCxFQUFnRCxZQUFXO0FBQ3pELE9BQUcsSUFBSCxFQUFTLDBDQUFUO0FBQ0QsR0FGRCIsImZpbGUiOiJlbWJlci1tZXRhbC9hcnJheS5qc2hpbnQuanMiLCJzb3VyY2VzQ29udGVudCI6WyJtb2R1bGUoJ0pTSGludCAtIGVtYmVyLW1ldGFsJyk7XG50ZXN0KCdlbWJlci1tZXRhbC9hcnJheS5qcyBzaG91bGQgcGFzcyBqc2hpbnQnLCBmdW5jdGlvbigpIHsgXG4gIG9rKHRydWUsICdlbWJlci1tZXRhbC9hcnJheS5qcyBzaG91bGQgcGFzcyBqc2hpbnQuJyk7IFxufSk7XG4iXX0=
+enifed("ember-metal/binding", [], function () {});
 //# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IiIsImZpbGUiOiJlbWJlci1tZXRhbC9iaW5kaW5nLmpzIiwic291cmNlc0NvbnRlbnQiOltdfQ==
-enifed('ember-metal/binding.jshint', ['exports'], function (exports) {
+enifed('ember-metal/binding.jshint', ['module'], function (module) {
   'use strict';
 
   module('JSHint - ember-metal');
   test('ember-metal/binding.js should pass jshint', function () {
     ok(true, 'ember-metal/binding.js should pass jshint.');
   });
 });
-//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImVtYmVyLW1ldGFsL2JpbmRpbmcuanNoaW50LmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7OztBQUFBLFFBQU0sQ0FBQyxzQkFBc0IsQ0FBQyxDQUFDO0FBQy9CLE1BQUksQ0FBQywyQ0FBMkMsRUFBRSxZQUFXO0FBQzNELE1BQUUsQ0FBQyxJQUFJLEVBQUUsNENBQTRDLENBQUMsQ0FBQztHQUN4RCxDQUFDLENBQUMiLCJmaWxlIjoiZW1iZXItbWV0YWwvYmluZGluZy5qc2hpbnQuanMiLCJzb3VyY2VzQ29udGVudCI6WyJtb2R1bGUoJ0pTSGludCAtIGVtYmVyLW1ldGFsJyk7XG50ZXN0KCdlbWJlci1tZXRhbC9iaW5kaW5nLmpzIHNob3VsZCBwYXNzIGpzaGludCcsIGZ1bmN0aW9uKCkgeyBcbiAgb2sodHJ1ZSwgJ2VtYmVyLW1ldGFsL2JpbmRpbmcuanMgc2hvdWxkIHBhc3MganNoaW50LicpOyBcbn0pO1xuIl19
-enifed("ember-metal/streams/simple", ["exports"], function (exports) {
-  "use strict";
-});
+//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImVtYmVyLW1ldGFsL2JpbmRpbmcuanNoaW50LmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7OztBQUFBLFNBQU8sc0JBQVA7QUFDQSxPQUFLLDJDQUFMLEVBQWtELFlBQVc7QUFDM0QsT0FBRyxJQUFILEVBQVMsNENBQVQ7QUFDRCxHQUZEIiwiZmlsZSI6ImVtYmVyLW1ldGFsL2JpbmRpbmcuanNoaW50LmpzIiwic291cmNlc0NvbnRlbnQiOlsibW9kdWxlKCdKU0hpbnQgLSBlbWJlci1tZXRhbCcpO1xudGVzdCgnZW1iZXItbWV0YWwvYmluZGluZy5qcyBzaG91bGQgcGFzcyBqc2hpbnQnLCBmdW5jdGlvbigpIHsgXG4gIG9rKHRydWUsICdlbWJlci1tZXRhbC9iaW5kaW5nLmpzIHNob3VsZCBwYXNzIGpzaGludC4nKTsgXG59KTtcbiJdfQ==
+enifed("ember-metal/streams/simple", [], function () {});
 //# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IiIsImZpbGUiOiJlbWJlci1tZXRhbC9zdHJlYW1zL3NpbXBsZS5qcyIsInNvdXJjZXNDb250ZW50IjpbXX0=
-enifed('ember-metal/streams/simple.jshint', ['exports'], function (exports) {
+enifed('ember-metal/streams/simple.jshint', ['module'], function (module) {
```

Comment:
We discussed this in slack, but just leaving a note here also: this is definitely an issue we have to address.  The `module` usage below is assuming a global module and shouldn't have been automatically wrapped with a non-existant import.